liquibase-integration-tests/src/test/resources/harness/docker/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     volumes:
     - "./mysql-init.sql:/docker-entrypoint-initdb.d/mysql-init.sql"
 
-  postgres:
+  postgre:
```

Comment:
@KushnirykOleh, let's keep this as `postgres`

Reply:
it was improper name refactoring, returned back

Reply:
Thank you @KushnirykOleh